schema/core.md

```diff
@@ -607,57 +609,66 @@ are used for a particular study.
 Note: `hasPart` and `isPartOf` are generally used for records that are grouped
 together or records that belong to other groupings. For example, `hasPart` may
 be used to indicate a record that consists of one or more additional
-CreativeWork (this is similar to a HydroShare collection). In contrast,
+`CreativeWork` (this is similar to a HydroShare collection). In contrast,
 `isPartOf` is used to indicate catalog records that reference a designated
-CreativeWork. As such, these are not demonstrated in the example below.
+`CreativeWork`. 
 
+The example below shows a simple schema for a `CreativeWork` that links to another
+`CreativeWork`.
 
 ``` json
 {
+  "@type": "CreativeWork",
+  "name": "Collection of Sample Datasets for IGUIDE",
+  "description": "A collection of sample datasets used by the I-GUIDE team",
+  "identifier": "https://my-unique-url.com/data-collection",
   "hasPart": {
-    "@type": "Dataset",
+    "@type": "CreativeWork",
     "name": "IGUIDE Shapefile Testing Resource",
-    "description": "Test HydroShare resource for I-GUIDE",
-    "url": "https://www.hydroshare.org/resource/9d413b9d57824a79b8239a5f7c4fdf51/data/contents/HUC6_Harvey_TxLaMsAr.shp?zipped=true&aggregation=true"
+    "description": "Test HydroShare resource for I-GUIDE - Shapefile",
+    "identifier": "https://my-unique-url.com/shapefile"
   }
 }
 ```
 
-Multiple relations can be defined using a list.
+For the example above, multiple relations can be defined using a list.
 
 ``` json
 {
+  "@type": "CreativeWork",
+  "name": "Collection of Sample Datasets for IGUIDE",
+  "description": "A collection of sample datasets used by the I-GUIDE team",
+  "identifier": "https://my-unique-url.com/data-collection",
   "hasPart": [
     {
       "@type": "CreativeWork",
       "name": "IGUIDE Shapefile Testing Resource",
       "description": "Test HydroShare resource for I-GUIDE - Shapefile",
-      "url": "https://my-unique-url.com/shapefile",
+      "identifier": "https://my-unique-url.com/shapefile",
```

Comment:
See comments above about identifiers. Also - is there a reason why these parts can't have both URL and identifier properties?

Reply:
Both should be valid.

Reply:
Actually, both properties are required. In most cases, they might be the same, but we could have a record for which the URL differs from the identifier, e.g., a published record with a URL and a DOI as an identifier.

Reply:
@igarousi - yes, they could be different. See the Science on Schema.org guidance for encoding identifiers. If the identifier is a DOI, there's a pretty specific way they suggest encoding it, which is more than just listing a DOI encoded as a URL. Also - I think it is Science on Schema.org that suggests that the URL for a record be the URL to a landing page for that record. That may get a little tricky for us as some records may have landing pages in a system like HydroShare, but others that are not stored in a repository may not - and we are also planning landing page functionality for the Catalog, so maybe the "url" property of a record should point at the I-GUIDE catalog landing page?

Reply:
Just to clarify, does that mean each record will have a landing page on the IGUIDE platform?

Reply:
@igarousi - yes, that is plan. Regardless of whether an iGUIDE catalog record has a landing page in a separate repository (e.g., HydroShare), it will still have a landing page in the iGUIDE catalog.